### ISSUE TITLE
[CLI] Chapters should have one line per language

### ISSUE THREAD

**Issue by MelkorLord** [MediaArea/MediaInfo#226]
The current display of MediaInfo for chapters, be it in default mode or XML/JSON, makes it very hard to parse in a sane way. If chapter titles have multiple languages, it becomes a real PITA to get them right.

Command : `mediainfo --full --language=raw file.mkv`

```
00:00:00.080                     : fr:Chapitre 01 - en:Chapter 01
00:01:03.920                     : fr:Chapitre 02 - en:Chapter 02
00:04:17.360                     : :No lang title
```

In this kind of example, we have FR/EN chapter titles as well as undefined language. This is very hard to parse in a sane way. We can not split the string with " - " because it can be present in the title itself for very good reasons (ie: "Introduction - The trigger to war") and we can not split reliably on 2-chars + colon because of the "undefined" case.

The above chapters list would be better displayed in the following way I think : 

```
00:00:00.080                     : Chapitre 01
00:00:00.080_fr                  : Chapitre 01
00:00:00.080_en                  : Chapter 01
00:01:03.920                     : Chapitre 02
00:01:03.920_fr                  : Chapitre 02
00:01:03.920_en                  : Chapter 02
00:04:17.360                     : No lang title
```

**Line 1** : Displays the first title in list regardless of the language, like the "default" entry. This keeps the compatibility with the current display
**Line 2 and 3** : Specific language title entry
**Last line** : The title but **without** the colon prefix, as in "Line 1"

What's your opinion?

**Comment by paxter** [comment]
This is still an unresolved problem for the JSON output.

As a solution, I would prefer to write each language into its own object.

So instead of

```json
{
        "@type": "Menu",
        "extra": {
          "_00_00_00_000": "fr:Chapitre 01 - en:Chapter 01",
          "_00_02_58_303": "fr:Chapitre 02 - en:Chapter 02",
          "_00_07_23_985": "fr:Chapitre 03 - en:Chapter 03",
          [...]
        }
}
```

maybe something like


```json
{
        "@type": "Menu",
        "extra": {
          "_00_00_00_000": {"fr": "Chapitre 01", "en": "Chapter 01"},
          "_00_02_58_303": {"fr": "Chapitre 02", "en": "Chapter 02"},
          "_00_07_23_985": {"fr": "Chapitre 03", "en": "Chapter 03"},
          [...]
        }
}
```

Since this is an open issue from 2018, I guess it's not a really important feature request. For the meantime, I parse that stuff by a mix of regex and string stripping. This is no perfect solution but should work in most cases. Nevertheless, a clean JSON output would be the golden solution.
